modules/test/playwright/tests/account-admin-web/accountRole.spec.ts

```diff
@@ -0,0 +1,241 @@
+import {loginTest} from '../../fixtures/loginTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+import getRandomString from '../../utils/getRandomString';
+import performLogin, {performLogout, userData} from '../../utils/performLogin';
+import getPageDefinition from '../layout-content-page-editor-web/utils/getPageDefinition';
+import getWidgetDefinition from '../layout-content-page-editor-web/utils/getWidgetDefinition';
+
+export const test = mergeTests(
+	accountsPagesTest,
+	applicationsMenuPageTest,
+	commercePagesTest,
+	dataApiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	loginTest()
+);
+
+test('LPD-28220 Can user with account manager role view and manage channel defaults', async ({
+	accountManagementWidgetPage,
+	accountsPage,
+	apiHelpers,
+	page,
+}) => {
+	const userAccount = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	userData[userAccount.alternateName] = {
+		name: userAccount.givenName,
+		password: 'test',
+		surname: userAccount.familyName,
+	};
+
+	const site = await apiHelpers.headlessSite.createSite({
+		name: 'Site' + getRandomInt(),
+	});
+
+	apiHelpers.data.push({id: site.id, type: 'site'});
+
+	const layout = await apiHelpers.headlessDelivery.createSitePage({
+		pageDefinition: getPageDefinition([
+			getWidgetDefinition({
+				id: getRandomString(),
+				widgetName:
+					'com_liferay_account_admin_web_internal_portlet_AccountEntriesManagementPortlet',
+			}),
+		]),
+		siteId: site.id,
+		title: getRandomString(),
+	});
+
+	try {
+		const organization =
+			await apiHelpers.headlessAdminUser.postOrganization({
+				name: 'Organization' + getRandomInt(),
+			});
+
+		const accounts = [];
+
+		const account1 = await apiHelpers.headlessAdminUser.postAccount({
+			name: getRandomString(),
+			type: 'business',
+		});
+
+		accounts.push(account1);
+
+		apiHelpers.data.push({id: account1.id, type: 'account'});
+
+		const account2 = await apiHelpers.headlessAdminUser.postAccount({
+			name: getRandomString(),
+			type: 'business',
+		});
+
+		accounts.push(account2);
+
+		apiHelpers.data.push({id: account2.id, type: 'account'});
+
+		await apiHelpers.headlessAdminUser.assignAccountToOrganization(
+			account1.id,
+			organization.id
+		);
+
+		await apiHelpers.headlessAdminUser.assignAccountToOrganization(
+			account2.id,
+			organization.id
+		);
+
+		await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+			account1.id,
+			[userAccount.emailAddress]
+		);
+
+		await apiHelpers.headlessAdminUser.assignUserToOrganizationByEmailAddress(
+			organization.id,
+			userAccount.emailAddress
+		);
+
+		await apiHelpers.jsonWebServicesUser.assignUsersToSite(
+			site.id,
+			userAccount.id
+		);
+
+		await performLogout(page);
+		await performLogin(page, userAccount.alternateName);
+
+		await page.goto(`/web/${site.name}/${layout.friendlyUrlPath}`);
+
+		const account1Cell = page.getByRole('cell', {
+			exact: true,
+			name: account1.name,
+		});
```

Comment:
this is a locator, should go in a page as a dynamic locator (a function that accept the account name)